docs/rackhd/skus.rst

```diff
@@ -270,13 +270,13 @@ package may have a static and template directory.
 .. code-block:: shell
 
     tar tzf pack.tar.gz:
-    config.json
-    static/
-    static/common/
-    static/common/discovery.overlay.cpio.gz
-    templates/
-    templates/ansible.pub
-    templates/esx-ks
+    config.json
+    static/
+    static/common/
+    static/common/discovery.overlay.cpio.gz
+    templates/
+    templates/ansible.pub
+    templates/esx-ks
```

Comment:
What happens to this file ? Looks like nothing changed. Or something wrong with the diff ?